src/lib/resolver.js

```diff
@@ -130,8 +131,9 @@ function resolveSelector(ctx, lang, args, expr, index) {
     return selector;
   }
 
-  const argValue = index[0].args ?
-    resolveIdentifier(ctx, lang, args, index[0].args[0].name)[1] : undefined;
+  const argName = index[0].args ? index[0].args[0].name : index[1];
+  const argValue = argName ?
+    resolveIdentifier(ctx, lang, args, argName)[1] : undefined;
```

Comment:
Wait, there's something I don't understand.  I did for the resolver into src/bindings/gaiabuild/legacy/resolver.  I'm thinking about this with a fresh mind and I don't see why this PR should be needed.  It does solve https://bugzil.la/1182568 though.  I need to investigate more.